src/Microsoft.Azure.CosmosEventSourcing/ChangeFeed/EventSourcingProvider.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.CosmosRepository.ChangeFeed;
+using Microsoft.Azure.CosmosRepository.ChangeFeed.Providers;
+
+namespace Microsoft.Azure.CosmosEventSourcing.ChangeFeed;
+
+public class EventSourcingProvider : IChangeFeedContainerProcessorProvider
```

Comment:
```suggestion
internal class EventSourcingProvider : IChangeFeedContainerProcessorProvider
```

Does the implementation need to be public?